packages/read-model/src/Product/Listed/CreateProductViewsFromEntityHelper.php

```diff
@@ -0,0 +1,71 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Shopsys\ReadModelBundle\Product\Listed;
+
+use Shopsys\FrameworkBundle\Model\Product\Product;
+use Shopsys\ReadModelBundle\Image\ImageViewFacade;
+use Shopsys\ReadModelBundle\Product\Action\ProductActionViewFacade;
+
+class CreateProductViewsFromEntityHelper
```

Comment:
Sadly, I agree... This class is really weird. 
I wanted to avoid DRY, because this method is used in `ListedProductVariantsViewFacade`, `ListedProductViewElasticFacade` and `ListedProductViewFacade` and I did not want to add it into factory (factory/facade dependency described in another comment).

I had an idea to create an abstract parent with this method, what do you think?

Reply:
I move the method to the factory

Reply:
implemented in dec7467, b5f7a91 and a1e073b